core.yaml

```diff
@@ -73,10 +75,6 @@ services:
     isShared: true
     svc: otomi-console
     paths: [/]
-    authz:
```

Comment:
Is it some old dead code that you are removing?

Reply:
The `authz` configuration in `core.yaml` is actually only used for the capability to disable passthrough of the JWT to the service endpoints.  Istio lets us do that via their RequestAuthentication resource, which we only create for services that get confused when it finds a bearer token that it is not responsible for. (Look at grafana.) So we need to strip JWTs for those workloads.

Reply:
So it only creates overhead when a `RequestAuthentication` is created with `forwardOriginalToken` set to `true`.

Thinking about it, we could remove the forwardOriginalToken and always set it to false, but I was envisioning more options later down the line. For now we should just remove that in core I think, and open it back up when there comes a need for it.